src/scripts.js

```diff
@@ -67,6 +68,12 @@ function cardButtonConditionals(event) {
     recipeToCookCard(event);
   }
 }
+
+function checkKeyPressed(event) {
+  if (event.keyCode == "13" && event.target.classList.contains('card-picture')) {
+    displayDirections(event);
+  }
+}
```

Comment:
To understand this line of code, is it when a user is on an image, they can click enter and the recipe information will display?

Reply:
Exactly. It was the cleanest way I could figure out how to make it so the event could be triggered when pressing enter or on click. For whatever reason, it didn't work to just wrap it in a button.

Reply:
That is a good addition to accessibility where users can navigate the product without the need of a mouse or trackpad.

Reply:
As a note, the other buttons all seem to work just fine on click and enter without needing this conditional.

Reply:
I'm glad you asked that because I had no idea what keycode was or what it was doing.  Looks like I learned something new about about javascript